cdap-ui/app/cdap/components/Home/HomeHeader/index.js

```diff
@@ -155,5 +157,6 @@ HomeHeader.propTypes = {
   sortOptions: PropTypes.array,
   activeSort: PropTypes.object,
```

Comment:
This should not be simple Object. It should define the shape of the object with exact properties that it will expect. Just object might not be too helpful in the future. [Reference](https://facebook.github.io/react/docs/reusable-components.html#prop-validation)